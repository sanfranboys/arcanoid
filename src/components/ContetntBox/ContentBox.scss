.form-component {
  display: flex;
  justify-content: center;
}
.form-component__wrapper {
  border: 4px solid var(--color-border-dark);
  text-align: center;
  padding: 30px 200px;
  min-width: 600px;
  box-shadow: 0 0 20px rgb(0 0 0 / 20%);
}
