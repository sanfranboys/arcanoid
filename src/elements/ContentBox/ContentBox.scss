.content-box {
  display: flex;
  justify-content: center;
}
.content-box__wrapper {
  border: 4px solid var(--color-border-dark);
  padding: 30px 100px;
  max-width: 600px;
  box-shadow: 0 0 20px rgb(0 0 0 / 20%);
}
