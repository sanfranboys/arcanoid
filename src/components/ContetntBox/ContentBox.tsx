import React, { FC } from 'react'
import './ContentBox.scss'

const ContentBox: FC = ({ children }) => (
  <div className="form-component">
    <div className="form-component__wrapper">{children}</div>
  </div>
)

export default ContentBox
