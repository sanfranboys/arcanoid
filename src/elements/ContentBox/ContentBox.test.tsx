import React from 'react'
import { create } from 'react-test-renderer'
import ContentBox from './ContentBox'

describe('ContentBox testing', () => {
  const props = {
    children: 'Content here',
  }

  describe('ContentBox snapshot testing', () => {
    it('snapshot matches', () => {
      const tree = create(<ContentBox {...props} />).toJSON()
      expect(tree).toMatchSnapshot()
    })
  })
})
