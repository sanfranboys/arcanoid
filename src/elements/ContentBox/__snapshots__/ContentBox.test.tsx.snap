// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContentBox testing ContentBox snapshot testing snapshot matches 1`] = `
<div
  className="content-box"
>
  <div
    className="content-box__wrapper"
  >
    Content here
  </div>
</div>
`;
